docs/multi-agent-manager.md

```diff
@@ -22,59 +30,79 @@ We first add a couple of finished AEA project:
+FET_CONNECTION_PRIVATE_KEY_PATH_STATION = Path(KEYS_DIR, weather_station_name, "fetchai_connection_private_key.txt").absolute()
+FET_CONNECTION_PRIVATE_KEY_PATH_STATION.write_bytes(FET_CONNECTION_PRIVATE_KEY_STATION)
+
+CERT_PATH_STATION = Path(CERTS_DIR, weather_station_name, "conn_cert.txt").absolute()
+
+BUILD_PATH_STATION = Path(BUILD_DIR, weather_station_name).absolute()
+os.makedirs(BUILD_PATH_STATION)
 
-FET_CONNECTION_PRIVATE_KEY_1 = b"bf529acb2546e13615ef6004c48e393f0638a5dc0c4979631a9a4bc554079f6f"
-FET_CONNECTION_PRIVATE_KEY_PATH_1 = Path("fetchai_connection_private_key_1.txt")
-FET_CONNECTION_PRIVATE_KEY_PATH_1.write_bytes(FET_CONNECTION_PRIVATE_KEY_1)
+FET_PRIVATE_KEY_CLIENT = b"589839ae54b71b8754a7fe96b52045364077c28705a1806b74441debcae16e0a"
+FET_PRIVATE_KEY_PATH_CLIENT = Path(KEYS_DIR, weather_client_name, "fetchai_private_key.txt").absolute()
+os.makedirs(Path(KEYS_DIR, weather_client_name))
+FET_PRIVATE_KEY_PATH_CLIENT.write_bytes(FET_PRIVATE_KEY_CLIENT)
 
-FET_PRIVATE_KEY_2 = b"589839ae54b71b8754a7fe96b52045364077c28705a1806b74441debcae16e0a"
-FET_PRIVATE_KEY_PATH_2 = Path("fetchai_private_key_2.txt")
-FET_PRIVATE_KEY_PATH_2.write_bytes(FET_PRIVATE_KEY_2)
+FET_CONNECTION_PRIVATE_KEY_CLIENT = b"c9b38eff57f678f5ab5304447997351edb08eceb883267fa4ad849074bec07e4"
+FET_CONNECTION_PRIVATE_KEY_PATH_CLIENT = Path(KEYS_DIR, weather_client_name, "fetchai_connection_private_key.txt").absolute()
+FET_CONNECTION_PRIVATE_KEY_PATH_CLIENT.write_bytes(FET_CONNECTION_PRIVATE_KEY_CLIENT)
 
-FET_CONNECTION_PRIVATE_KEY_2 = b"c9b38eff57f678f5ab5304447997351edb08eceb883267fa4ad849074bec07e4"
-FET_CONNECTION_PRIVATE_KEY_PATH_2 = Path("fetchai_connection_private_key_2.txt")
-FET_CONNECTION_PRIVATE_KEY_PATH_2.write_bytes(FET_CONNECTION_PRIVATE_KEY_2)
+CERT_PATH_CLIENT = Path(CERTS_DIR, weather_client_name, "conn_cert.txt").absolute()
+
+BUILD_PATH_CLIENT = Path(BUILD_DIR, weather_client_name).absolute()
+os.makedirs(BUILD_PATH_CLIENT)
 ```
 
 Add the agent instances
 ``` python
 agent_overrides = {
-    "private_key_paths": {"fetchai": str(FET_PRIVATE_KEY_PATH_1.absolute())},
-    "connection_private_key_paths": {"fetchai": str(FET_CONNECTION_PRIVATE_KEY_PATH_1.absolute())}
+    "private_key_paths": {"fetchai": str(FET_PRIVATE_KEY_PATH_STATION)},
+    "connection_private_key_paths": {"fetchai": str(FET_CONNECTION_PRIVATE_KEY_PATH_STATION)}
 }
 
-public_id = PublicId.from_str("fetchai/p2p_libp2p:0.14.0")
+p2p_public_id = PublicId.from_str("fetchai/p2p_libp2p:0.14.0")
 
 component_overrides = {
-    **public_id.json,
+    **p2p_public_id.json,
     "type": "connection",
+    "build_directory": str(BUILD_PATH_STATION),
     "cert_requests": [{
       "identifier": "acn",
       "ledger_id": "fetchai",
       "not_after": '2022-01-01',
       "not_before": '2021-01-01',
       "public_key": "fetchai",
-      "save_path": f"{weather_station_id.author}/{weather_station_id.name}/.certs/conn_cert.txt"
+      "save_path": str(CERT_PATH_STATION)
```

Comment:
Both overrides are currently not processed by the build and issue_certificates calls.